cnxeasybake/oven.py

```diff
@@ -171,11 +171,9 @@ def update_css(self, css_in=None, clear_css=False):
                     #            <IdentToken html>,
                     #            <WhitespaceToken>,
                     #            <StringToken "http://www.w3.org/1999/xhtml">]
-                    if ((len(rule.prelude) == 4 and
-                         rule.prelude[0].type == 'whitespace' and
-                         rule.prelude[1].type == 'ident' and
-                         rule.prelude[2].type == 'whitespace' and
-                         rule.prelude[3].type == 'string')):
+                    if len(rule.prelude) == 4 and \
+                           [tok.type for tok in rule.prelude] == \
+                           ['whitespace', 'ident', 'whitespace', 'string']:
```

Comment:
woah, nice! Yeah, my code was suuper ugly and I didn't know how to do listcomps in python. Also, the `\` is nice in that it clarifies "this line is not done".